arran-runtime-blog-post/blog.md

```diff
@@ -0,0 +1,107 @@
+# Code it: creating hillshade effects using ArcGIS Runtime #
```

Comment:
Wondering if we want to duplicate the blog post in the repository.  Maybe we should consider including a readme.md file instead giving a brief overview of the purpose of the code and a link to the blog post?